### PR TITLE
Added S3 objects backlog tagger

### PR DESCRIPTION
**Pull request by victorskl**
* Added libs3 for reusable S3 operations
* Added notes about Cost cautious on S3 API requests

### CODE REVIEW

File: tagger.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+
+FILTER_SUFFIX = ".bam"
+
+if __name__ == '__main__':
+    """
+    Meant to run one time, offline tool to re/process tagging as implemented in services.tag_s3_object(..)
+    
+    Cost:
+    1. Listing bucket is paginated 1000 objects per request, see libs3.get_matching_s3_keys(..), therefore
```
Comment (brainstorm) [inline tagger.py]: @victorskl Did you check if there was a parameter in Boto3 to bundle more paginated objects per request?

Comment (victorskl) [reply tagger.py]: Yes; 1000 objects is MAX.
 https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.list_objects_v2

File: tagger.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        450,000 objects * $0.0055 / 1000 = $2.475
+    2. Tagging an object requires 1 GET request and 1 PUT request, therefore, say 2712 .bam out of 450,000 
+        (2712 * $0.0055) PUT + (2712 * $0.00044) GET = $14.916 + $1.19328 = $16.10928
```
Comment (reisingerf) [inline tagger.py]: Actually:
450,000 objects at 1000 objects per request = 450 LIST requests
450 requests at $0.0055 per 1000 requests = $0.002475

Similarly:
1 PUT + 1 GET request at $0.0055 and $0.0044 per 1000 requests = $0.00594 per 1000 requests
For 2712 objects thats 2712 * 0.00594 / 1000 = 0.01610928

negligible cost in any case ;-)

Comment (victorskl) [reply tagger.py]: Thanks Florian. Updated the note.

**Pull request merged**